utils/constants.js

```diff
@@ -20,12 +20,12 @@ export const daysOfWeek = [
 export const modesOfTransport = [
   "Bicycle",
   "Bus",
+  "Carpool",
+  "Motorbike",
+  "Taxi Ride",
```

Comment:
We want to be careful about changing `Taxi / ride share` to `Taxi Ride`

`Taxi / ride share` means:
- using taxi services to commute
- using ride sharing services like Uber to commute (there might be others)

changing to 'Taxi Ride' effectively excludes ride sharing services as an option.

I prefer we not shorten the descriptions of transport method selections out of 'convenience' to us, unless the shorter description still captures the same meaning of the original description.

Reply:
This has been fixed now. Today I did another commit let the text ramain Taxi/Ride Share matching the figma.